Stripe/PublicHeaders/STPAddress.h

```diff
@@ -221,6 +247,26 @@ typedef NS_ENUM(NSUInteger, STPBillingAddressFields) {
  */
 + (PKAddressField)applePayAddressFieldsFromBillingAddressFields:(STPBillingAddressFields)billingAddressFields;
 
+/**
+ Converts a set of STPContactField values into the closest equivalent
+ representation of PKAddressField options
+
+ @param contactFields Stripe contact fields values to convert.
+ @return The closest represenation of the contact fields as
+ a PKAddressField value.
+ */
++ (PKAddressField)pkAddressFieldsFromStripeContactFields:(nullable NSSet<STPContactField> *)contactFields;
+
+/**
+ Converts a set of STPContactField values into the closest equivalent
+ representation of PKContactField options
+
+ @param contactFields Stripe contact fields values to convert.
+ @return The closest represenation of the contact fields as
+ a PKContactField value.
+ */
++ (NSSet<PKContactField> *)pkContactFieldsFromStripeContactFields:(nullable NSSet<STPContactField> *)contactFields API_AVAILABLE(ios(11.0));
```

Comment:
looks like this `API_AVAILABLE` guard isn't working – I think it needs to be `API_AVAILABLE(ios(11))`